### PR TITLE
LPS-134215 Fix normalize dataType

### PR DESCRIPTION
**Pull request by interaminense**

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Validation/transform.js [hunk 1]
```diff
@@ -73,18 +60,29 @@ const getValidation = (
...
 	};
 };
 
+export const normalizeDataType = (initialDataType) => {
+	if (initialDataType === 'double' || initialDataType === 'integer') {
+		return 'numeric';
+	}
+
+	return initialDataType;
+};
```
Comment (interaminense) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Validation/transform.js:78]:
```suggestion
export const normalizeDataType = (initialDataType) => {
	return initialDataType === 'double' || initialDataType === 'integer' ? 'numeric' : initialDataType;
};
```

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Validation/transform.js [hunk 2]
```diff
@@ -73,18 +60,29 @@ const getValidation = (
...
+
+export const getLocalizedValue = ({defaultLanguageId, editingLanguageId}) => {
+	return (value) => {
+		return value[editingLanguageId] || value[defaultLanguageId];
+	};
+};
```
Comment (interaminense) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Validation/transform.js:84]:
```suggestion
export const getLocalizedValue = ({defaultLanguageId, editingLanguageId}) => (value) => value[editingLanguageId] || value[defaultLanguageId];
```

**Comment by natocesarrego** [comment]
@mateusgomes01, is this pull still necessary? If not, can you please close it?

**Comment by carolmariaabb** [comment]
Resent https://github.com/liferay-forms/liferay-portal/pull/719

**Pull request closed without merging**